### PR TITLE
recreating the hierarchy endpoint from redis

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

File: api/v1/helpers/nouns/subjects.js [hunk 1]
```diff
@@ -257,7 +317,7 @@ function traverseHierarchy(res) {
   }
 
   // return 0 only if both filteredChildrenArr.length and pruneNode return zero
-  return pruneNode(res) || filteredChildrenArr.length;
+  return pruneNodeV2(res) || filteredChildrenArr.length;
```
Comment (pallavi2209) [inline api/v1/helpers/nouns/subjects.js]: @shriramshankar Do you need this line change?

**Comment by shriramshankar** [comment]
Just added some changes to this branch that attaches aspects back into the sample object. The perspective page on Salesforce.SFDC_Core loads in a flash!!

@pallavi2209  you might want to run your tests against this change.

**Comment by pallavi2209** [comment]
@shriramshankar Thats great!
Is it possible to use a toggle for these changes? I want to compare the response time with db vs redis, and a toggle would be great to have in that scenario.

**Comment by iamigo** [comment]
@shriramshankar close?

**Pull request closed without merging**